app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApplicationController < ActionController::API
+  protect_from_forgery
```

Comment:
[ Required ] Please check on this method it appears to be failing when the `http://127.0.0.1:3000/api/v1/doctors` route is visited, I think it would be better if we remove it.